### DISCUSSION TITLE
Welcome to the Virtual Coffee Contributors Discussion Board!

### DISCUSSION THREAD

**Discussion by danieltott** [Virtual-Coffee/VC-Community-Docs#128] [General]
Hello, and welcome to the Virtual Coffee Contributors Discussion board! As part of our Contributors community, you have access to Discussion Boards: message boards here on GitHub where the majority of our communications take place. To help you get the most out of discussion boards we wanted to create a short welcome message, as well as [guide to using the VC Contributors Discussion Board](https://github.com/Virtual-Coffee/VC-Community-Docs/blob/main/docs/vc-discussion-boards.md)

## Why did we choose to use discussion boards?

The Virtual Coffee maintainers spent quite a bit of time trying to decide the best possible way for all contributors to have discussions. It was important to us that the medium be:

- Relatively straightforward to get into
- Accessible for our contributors
- Permanent so we could always reference earlier conversations
- Flexible enough to work with the various types of communications a volunteer group like this need (ad hoc, q&a, open discussions, etc)

Seeing as we are already using GitHub for hosting our documentation and coordinating action items through issues, having conversations happen in discussions felt like the natural choice. This will also replace the team discussions that we experimented with for a bit.

Please take a moment to read our [guide to using the VC Contributors Discussion Board](https://github.com/Virtual-Coffee/VC-Contributors/blob/main/docs/vc-discussion-boards.md) - and special thanks to @tkshill for putting that together ❤️ 

## Feedback

If you have any questions after reading this, please do not hesitate to reply to this discussion, [post a new Question discussion topic](https://github.com/Virtual-Coffee/VC-Community-Docs/discussions/new), ask a Virtual Coffee Maintainer about it, or email us at hello@virtualcoffee.io.

Thank you!

**Comment by bogdaaamn** [comment]
Hey friends <3

**Comment by BekahHW** [comment]
@Virtual-Coffee/coffee-talk-team - Just testing here to see if our team is notified.

**Reply by bogdaaamn** [reply]
I did get a notification

**Reply by BekahHW** [reply]
Thanks, @BogDAAAMN

**Reply by aurelieverrot** [reply]
Me too

**Comment by nickytonline** [comment]
👋
